internal/formats/spdx22json/to_format_model.go

```diff
@@ -153,6 +153,7 @@ func toFiles(s sbom.SBOM) []model.File {
 				},
 				// required, no attempt made to determine license information
 				LicenseConcluded: "NOASSERTION",
+				// LicenseConcluded: "NONE", 19 changes
```

Comment:
Why was this inserted?

Reply:
ops, it's a leftover from output comparison, clean it up